widgets/src/main/java/ru/startandroid/widgets/adapter/WidgetAdapter.kt

```diff
@@ -55,7 +57,7 @@ constructor(
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): WidgetContainerHolder {
         val widgetContent = widgetProvider.getWidget(viewType)
         val view = LayoutInflater.from(parent.context).inflate(R.layout.widget_container, parent, false)
-        return WidgetContainerHolder(view, widgetContent, widgetAdapterCallback)
+        return WidgetContainerHolder(view, widgetContent, widgetAdapterCallback,context)
```

Comment:
You don't need to pass context here, because there is parent.context. I use it in previous string.
If you have a view, you have a context )